draft-specifications/CapTP Specification.md

```diff
@@ -492,13 +492,16 @@ parties MUST send upon a new connection. The operation looks like this:
                   acceptable-location-sig>  ; CapTP signature
 ```
 
-It's important that we only have one bidirectional connection between a CapTP
-session. Before trying to connect to a peer, it's important to check that a
-connection is not already open before proceeding. There is a race condition
-called the "crossed hellos" problem whereby each side tries to open a new
-connection at the same time and only one of these connection attempts must
-succeed. The way to check which attempt should succeed is to sort the serialized
-`session-pubkey`, the highest of the two wins.
+An important aspect of CapTP is that only one active session between two peers
+should exist. This allows peers to equality checks against objects from a remote
```

Comment:
Missing a verb here. Did you mean "This allows peers to *perform* equality checks..."?